src/PosthogAnalytics.ts

```diff
@@ -124,7 +124,10 @@ export class PosthogAnalytics {
         respect_dnt: true,
         advanced_disable_decide: true,
       });
-      this.enabled = true;
+      if (PosthogAnalytics.getPlatformProperties().matrixBackend === "jssdk") {
```

Comment:
I'd suggest moving this to the `if (posthogConfig.project_api_key && posthogConfig.api_host)` to not initialize Posthog in widgets either